server/routes/user.routes.js

```diff
@@ -278,7 +278,7 @@ router.post('/upload-image', authenticate, Upload.uploadImage, UploadController.
 
 /**
  * @swagger
- * /user/all:
+ * /user/all/:limit/:count:
```

Comment:
Pagination should not affect the current endpoint. Let's make this query params instead of including it in the endpoint.

```suggestion
 * /user/all
```